src/io/saveload.jl

```diff
@@ -34,12 +34,12 @@ function save(filename::AbstractString, x::AbstractObject)
     return nothing
 end
 
```

Comment:
This can be later, but should also simplify variable names (and other func): 
* allunderlying -> underlying
* allhistory ->history
* gethistory ->get_history()

Reply:
Good call, I'll make those changes.

Reply:
Done